app/person-escort-record/views/form-step.njk

```diff
@@ -1,15 +1,19 @@
 {% extends "form-wizard.njk" %}
 
+{% set mainClasses = "interruption-card" if options.stepType == 'interruption-card' %}
```

Comment:
Just a thought - is it worth having a `form-step` and `form-interruption-card` template?

Reply:
Not a bad thought. Although would mean updating the render method to handle changing of the template and probably now having 3 templates (one base to inherit from, plus one for the two _types_).

Maybe one to consider if the type becomes more drastically different in what it renders but in this case it's only applying a class and hiding the save action.

What do you think?